homeassistant/components/sensor/zwave.py

```diff
@@ -56,8 +60,17 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
         # Check workaround mappings for specific devices.
         if specific_sensor_key in DEVICE_MAPPINGS:
             if DEVICE_MAPPINGS[specific_sensor_key] == WORKAROUND_IGNORE:
+                logger.info('Ignoring device: %s, Index: %s',
+                            zwave._value_name(value), value.index)
                 return
 
+    if value.label == 'Previous Reading':
+        # Power measurement devices may export a 'Previous Reading' value
+        # This can be problematic, as can be seen here:
+        # https://github.com/home-assistant/home-assistant/issues/3509
+        logger.info('Ignoring device: %s', zwave._value_name(value))
```

Comment:
I'm glad you can confirm this :smiley: 
I would prefer if we made these methods public, because `value.label` omits the node name (e.g. manufacturer and device name)